stubs/Pillow/PIL/ImageTk.pyi

```diff
@@ -1,18 +1,53 @@
-from _typeshed import Incomplete
+import _tkinter
+import tkinter
+from _typeshed import ReadableBuffer, StrOrBytesPath, SupportsRead
 from typing import Any
 
+from PIL.Image import Image, _Box, _Mode, _Size
+
 class PhotoImage:
-    tk: Any
-    def __init__(self, image: Incomplete | None = ..., size: Incomplete | None = ..., **kw) -> None: ...
+    tk: _tkinter.TkappType
+    def __init__(
+        self,
+        image: Image | _Mode | None = None,
+        size: _Size | None = None,
+        *,
+        file: StrOrBytesPath | SupportsRead[bytes] = ...,
+        data: ReadableBuffer = ...,
+        # These are the same as tkinter.PhotoImage:
```

Comment:
Maybe we should add a comment to `tkinter/__init__.pyi` that reminds about keeping these in sync.